hyperlane-mailbox/tests/harness.rs

```diff
@@ -180,3 +210,51 @@ async fn test_latest_checkpoint() {
     // The index is 0-indexed
     assert_eq!(index, 0u32);
 }
+
+#[tokio::test]
+async fn test_process() {
+    let (mailbox, _id, recipient) = get_contract_instance().await;
```

Comment:
I assume you just hadn't gotten to it yet bc of the version issue but we need to deploy the default ISM too right

Reply:
umm the default ISM is set to the test ISM already